api/resources/translations/messages-en.properties

```diff
@@ -638,6 +638,9 @@ email.invalid = Invalid email address.
 empty = It looks like you sent an empty message, please try to resend. If you continue to have this problem please contact your supervisor.
 enketo.constraint.invalid = Value not allowed
 enketo.constraint.required = This field is required
+enketo.drawwidget.signature = signature
+enketo.drawwidget.drawing = drawing
+enketo.drawwidget.annotation = annotation
```

Comment:
I suggest reaching out to fellow teammates to get Spanish, French and Nepali translations so that we keep the translations in sync. 

I can provide the Swahili translations.

Reply:
The French translations from a French speaking colleague:
Signature (remains the same)
Drawing -> Dessiner
Annotation (remains the same)

Reply:
Swahili translations 
signature: sahihi
drawing: uchoraji
annotation: maelezo

Reply:
drawing could also be kuchora @Benmuiruri